dashboards/google-cloud-composer/cloud-composer-monitor.json

```diff
@@ -0,0 +1,389 @@
+{
+  "category": "CUSTOM",
+  "dashboardFilters": [
+    {
+      "filterType": "RESOURCE_LABEL",
+      "labelKey": "project_id",
+      "templateVariable": "Project"
+    },
+    {
+      "filterType": "RESOURCE_LABEL",
+      "labelKey": "environment_name",
+      "templateVariable": "Cluster"
+    }
+  ],
+  "displayName": "sb-data-projects-composer-load-test-v1",
+  "etag": "07c4608fd0b2623bfdb5c444238ba2fb",
```

Comment:
Remove eTag -- these are specific to your specific project and custom dashboard, and should be scrubbed before we publish as a public sample dashboard.

Reply:
done!!